examples/sample-data-generator/complex-instructions.yaml

```diff
@@ -0,0 +1,25 @@
+source:
+  ref: generate_sample_demo
+  fields:
+    - name_parts: [ email ]
+      type: varchar
+      required: true
+    - name_parts: [ gebruikersnaam ]
```

Comment:
I'm really impressed by the fact that OpenAI GPT4 is able to interpret mixed language instructions, though the audience of PACE probably is not familiar with the Dutch language, I'd propose to use English fields. I'm aware that your video already has these Dutch field names in it, though for the docs, it might be better to use English only.

Reply:
I can't really, because the csv demo is already using Dutch iban numbers!
I'll clarify in the docs